src/commands/users/list.ts

```diff
@@ -0,0 +1,41 @@
+import { runCommand } from '../../util/command';
+import { outputFormat } from './util';
+
+export = {
+	command: 'list',
+	desc: 'List all users',
+	builder: yargs => {
+		yargs.options({
+			'active': {
+				desc: 'Only retrieve active users',
+				type: 'boolean',
+				group: 'Search',
+				default: undefined
+			},
+
+			'offset': {
+				default: 0,
+				desc: 'Search offset',
+				type: 'number',
+				group: 'Search'
+			},
+
+			'limit': {
+				default: 20,
```

Comment:
Should we refactor this to a global default?

Reply:
will do